### PR TITLE
JWWEB-793 Field Preview for Transaction Details

### PR DESCRIPTION
**Pull request by Creepypoke**
We don’t do special cases of field preview. All business logic are move to Transaction Detail component

### CODE REVIEW

File: .idea/runConfigurations/All_unit_tests.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <node-options value="" />
     <jest-package value="$PROJECT_DIR$/node_modules/jest" />
     <working-dir value="$PROJECT_DIR$" />
+    <jest-options value="--watch" />
```
Comment (stcherenkov) [inline .idea/runConfigurations/All_unit_tests.xml]: What does this do?

File: src/components/FieldPreview/FieldPreview.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  const value = currentTarget.getAttribute('data-value')
+
+  if (value) {
+    clipboard.copyText(value)
+  }
+}
+
+function joinAddContact(body: string): string {
+  return body
+    ? `/contacts/add/address=${body}`
```
Comment (stcherenkov) [inline src/components/FieldPreview/FieldPreview.js]: Error: `/contacts/add?address=${body}`

If there is no body, better to leave link to `/contacts/add` than empty link

File: src/components/FieldPreview/FieldPreview.stories.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { storiesOf } from '@storybook/react'
+import {
+  withKnobs,
+  text,
+  boolean,
+} from '@storybook/addon-knobs'
+
+import { FieldPreview } from 'components/FieldPreview/FieldPreview'
+
+const PRIVATE_KEY = '0xfb27c2394586feb01403ba3643b519c8f209e0427b31f510a6877b494c020f59'
```
Comment (stcherenkov) [inline src/components/FieldPreview/FieldPreview.stories.js]: It's a public key :)

Comment (Creepypoke) [reply src/components/FieldPreview/FieldPreview.stories.js]: Yeah, I know, I just copy this line from other story

File: .idea/runConfigurations/All_unit_tests.xml (hunk 1 above)
Comment (Creepypoke) [reply .idea/runConfigurations/All_unit_tests.xml]: This flag turn on test runner in `watch` mode, that's means that all changes of test files rerun tests

**Pull request merged**